game/outfits.rpy

```diff
@@ -1221,6 +1227,12 @@ init -1 python in jn_outfits:
         unlocked=False,
         is_jn_wearable=True
     ))
+    __register_wearable(JNAccessory(
+        reference_name="jn_accessory_double_white_hairpins",
+        display_name="Double white hairpins",
```

Comment:
I would suggest to call them hairbands instead of hairpins, since all items at the same location of Natsuki's hair are called "hairbands".

Reply:
Good call - it would be more consistent. I'll get that updated